test/spec.yml

```diff
@@ -427,6 +427,30 @@
             request_id: *
         exit: 1
 
+- test: Test user code timeout (bad-path)
+  cleanup:
+    - bn remove timeoutforsure
+  steps:
+    -   in: bn create node8 timeoutforsure 
+    -   in: |-
+           cat > function.js << EOL
+           exports.handler = async (body, req) => {
+             const msleep = ms => new Promise(resolve => setTimeout(resolve, ms));
+             await msleep(100000);
+           };
+           EOL
+    -   in: bn deploy timeoutforsure
+    -   in: bn invoke timeoutforsure
+        err: |-
+            <html>
+            <head><title>504 Gateway Time-out</title></head>
```

Comment:
please open issues for all these unfriendly errors you encounter (these are generated by libra for example)

Reply:
I know it's a Libra error...